src/main/java/com/hcl/fsc/services/EmployeeServiceImpl.java

```diff
@@ -1,103 +1,509 @@
+import com.hcl.fsc.mastertables.OfferedBand;
+import com.hcl.fsc.mastertables.OfferedDesignation;
+import com.hcl.fsc.mastertables.OfferedSubBand;
+import com.hcl.fsc.mastertables.OnboardingStatus;
+import com.hcl.fsc.mastertables.Region;
+import com.hcl.fsc.mastertables.State;
+import com.hcl.fsc.mastertables.UgDegree;
+import com.hcl.fsc.mastertables.UgPg;
+import com.hcl.fsc.mastertables.UgSpecialization;
+import com.hcl.fsc.repositories.CollegeTieringRepository;
+import com.hcl.fsc.repositories.EmployeeRepository;
 import com.hcl.fsc.repositories.GenderRepository;
+import com.hcl.fsc.repositories.L1Repository;
+import com.hcl.fsc.repositories.L2Repository;
+import com.hcl.fsc.repositories.L3Repository;
+import com.hcl.fsc.repositories.L4Repository;
+import com.hcl.fsc.repositories.LobRepository;
+import com.hcl.fsc.repositories.LocationRepository;
+import com.hcl.fsc.repositories.OfferedBandRepository;
+import com.hcl.fsc.repositories.OfferedDesignationRepository;
+import com.hcl.fsc.repositories.OfferedSubBandRepository;
+import com.hcl.fsc.repositories.OnboardingStatusRepository;
+import com.hcl.fsc.repositories.RegionRepository;
+import com.hcl.fsc.repositories.StateRepository;
+import com.hcl.fsc.repositories.UgDegreeRepository;
+import com.hcl.fsc.repositories.UgPgRepository;
+import com.hcl.fsc.repositories.UgSpecializationRepository;
 
 
 @Service
 public class EmployeeServiceImpl {
 	
     @Autowired
-    private CandidateRepository candidateRepository;
+    private EmployeeRepository employeeRepository;
     
     @Autowired
     private GenderRepository genderRepository;
+    
+    @Autowired
+    private LobRepository lobRepository;
+    
+    @Autowired
+    private LocationRepository locationRepository;
+    
+    @Autowired
+    private RegionRepository regionRepository;
+    
+    @Autowired
+    private CollegeTieringRepository collegeTieringRepository;
+    
+    @Autowired
+    private StateRepository stateRepository;
+
+	@Autowired
+    private L1Repository l1Repository;
+	
+	@Autowired
+    private L2Repository l2Repository;
+
+	@Autowired
+    private L3Repository l3Repository;
+	
+	@Autowired
+    private L4Repository l4Repository;
+	
+	@Autowired
+    private UgDegreeRepository ugDegreeRepository;
+	
+	@Autowired
+    private UgPgRepository ugPgRepository;
+	
+	@Autowired
+    private UgSpecializationRepository ugSpecializationRepository;
+	
+	@Autowired
+    private OnboardingStatusRepository onboardingStatusRepository;
+	
+	@Autowired
+    private OfferedBandRepository offeredBandRepository;
+	
+	@Autowired
+    private OfferedSubBandRepository offeredSubBandRepository;
+    
+
+	@Autowired
+    private OfferedDesignationRepository offeredDesignationRepository;
+    
+    
+    public List getTable(String mastertable) {
+		return repo(mastertable).findAll();
+		
+    }
+    
+    public JpaRepository repo(String mastertable) {
+    	
+    	
+    	if(mastertable.equals("gender")) 
```

Comment:
Make sure this functionality not break if API request is passed as GENDER or Gender